meta/runtime.yml

```diff
@@ -1,5 +1,5 @@
 ---
-requires_ansible: ">=2.9.10"
+requires_ansible: ">=2.15.0"
```

Comment:
Let us bump the minimum version to 2.12 or 2.13 rather than directly to 2.15

Reply:
Allison _specifically_ asked for this change in https://redhat-internal.slack.com/archives/CE3UL7F8V/p1714068882350819?thread_ts=1714067370.332829&cid=CE3UL7F8V